src/test-server-manager.cpp

```diff
@@ -954,7 +954,7 @@ static void verify_proof(FinishEpochResponse &response, const Proof &proof, uint
         "Validity proof should have a valid notices epoch root hash");
     ASSERT(validity.has_machine_state_hash() && !validity.machine_state_hash().data().empty(),
         "Validity proof should have a valid machine state hash");
-    ASSERT(validity.keccak_in_hashes_siblings_size() > 0, "Proof should have keccak in hashes siblings");
+    ASSERT(validity.output_hash_in_output_hashes_siblings_size() > 0, "Proof should have keccak in hashes siblings");
```

Comment:
Fix error message. Are there other instances of the word "keccak" we might have missed?